definitions/modules.yaml

```diff
@@ -0,0 +1,36 @@
+# Modules for interpreted languages.
+
+name: PythonModules
+doc: Python module files.
+sources:
+# Setuptools allows multiple formats for python eggs.
+- type: FILE
+  attributes:
+    paths:
+    - '%%users.homedir%%/.local/lib/python2.*/*-packages/*.egg-info'
+    - '%%users.homedir%%/.local/lib/python*/*-packages/*.egg'
+    - '/usr/local/lib/python*/*-packages/*.egg-info'
+    - '/usr/local/lib/python*/*-packages/*.egg'
+  supported_os: [Linux]
+- type: DIRECTORY
+  attributes:
+    paths:
+    - '%%users.homedir%%/.local/lib/python2.*/*-packages/*.egg-info'
+    - '%%users.homedir%%/.local/lib/python*/*-packages/*.egg'
+    - '/usr/local/lib/python*/*-packages/*.egg-info'
+    - '/usr/local/lib/python*/*-packages/*.egg'
+  supported_os: [Linux]
+urls: ['https://pythonhosted.org/setuptools/formats.html']
```

Comment:
What about the non-egg formats? e.g. wheel https://wheel.readthedocs.org/en/latest/